AsyncDisplayKit/ASVideoNode.mm

```diff
@@ -286,6 +312,20 @@ - (void)play
     }];
   }
   
+  if (!_playerNode) {
```

Comment:
We need to factor out this code to reduce duplication / chance of it changing separately.  Maybe define a method -playerNode that creates and returns the playerNode if not created.  Do not add it as a subnode in that method, but instead have a separate add or insert call wherever that operation is needed.